src/main/java/cs/ualberta/cmput402/tictactoe/board/Board.java

```diff
@@ -11,6 +11,8 @@
     private Player currentPlayer;
     private Player winner;
     private Player board[][];
+    private int moveCount;
```

Comment:
I think it is better to reset moveCount in the initBoard() in order to make sure moveCount is reset to 0 every time a new game starts.